.github/CHANGELOG.md

```diff
@@ -56,6 +56,9 @@
 
 ### Bug fixes
 
+* Fix a bug in the `LM::multiQubitOp` kernel implementation.
```

Comment:
Could you please provide more context here?

Reply:
Will do.